source/_integrations/zwave_js.markdown

```diff
@@ -412,14 +412,16 @@ action:
 
 ### Device automations
 
-Z-Wave JS has support for device triggers and conditions. To use a device automation, use the automation UI to select the "Device" trigger/condition type, and then pick your Z-Wave JS device. Under trigger/condition types, you will see Z-Wave JS specific entries.
+Z-Wave JS has support for device triggers, conditions, and actions. To use a device automation, use the automation UI to select the "Device" trigger/condition/action type, and then pick your Z-Wave JS device. Under trigger/condition/action types, you will see Z-Wave JS specific entries.
```

Comment:
Maybe we should just remove the device automations section? This is standard for all integrations that support them. The user is guided in the GUI how to use them.

Reply:
Yeah I guess it doesn't really provide any useful information, it just states that it's available